src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -50,30 +55,35 @@ const DeleteDocumentForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Delete Document"
       state={state}
       statusType="Deletion status"
       defaultMessage="To be deleted..."
     >
-      <form onSubmit={handleDeleteDocument} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="delete-doctype">Select document type to delete: </label>
-          <DocumentSelection htmlId="delete-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Delete Document
-          </button>
-        </div>
-      </form>
+      <Box sx={{ minWidth: 120 }}>
+        <form onSubmit={handleDeleteDocument} autoComplete="off">
```

Comment:
Now that I think about it, I don't think there are text field for this form or for FetchDocumentForm, they're already pre-populated.

Think we can safely remove the autoComplete attributes without affecting anything.

Reply:
Removed.